CHANGELOG.md

```diff
@@ -4,11 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [3.3.0] - 2018-10-22
+## [3.3.0] - Unreleased
 
 ### Added
 - Add `InputVirtualizedSelect` input component for custom fields. #131
 
+### Changed
```

Comment:
Could these be moved to an unreleased header? We already released 3.3.0 with that `InputVirtualizedSelect` component.